### PR TITLE
[S264] Date Format bug fix

### PR DESCRIPTION
**Pull request by tienhoah**
# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
- date format bug in slider create inconsistent labeling.
- also fix the wrong date (from bug above) in download report.
- minor refactor.

## Type of change
- remove deleted files related to date format.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Instructions

Explain what someone needs to do in order to test the functionality of the changes.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tienhoah** [comment]
[Date Format bug fix](https://trello.com/c/RjRICLHX/264-date-format-bug-fix)

**Review by gridcell — APPROVED** [review]

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -37,10 +36,8 @@ const Footer = () => {
   const mapLayers = new MapLayers()
   const { isAuthenticated } = useAuth0()
 
-  const downloadReport = () => {
-    const date = dateFormat(dateContext)
-
-    mapLayers.downloadReport(countryContext, date)
+  const downloadMapLayerReport = () => {
```
Comment (saanobhaai) [inline src/components/Footer/index.js]: This name isn't really accurate -- I'd keep it just `downloadReport`

File: src/components/DateSlider/index.js [hunk 2]
```diff
@@ -131,10 +121,7 @@ const DateSlider = () => {
   useEffect(() => {
     if (!(countryContext === null || countryContext === '')) {
       availableDate.getDate(countryContext, species).then(date => {
-        const dateData = date.data
-
-        const dateFormat =
-          dateData.length > 0 ? dateMarksFormat(dateData) : dateData
+        const dateFormat = dateMarksFormat(date.data)
```
Comment (saanobhaai) [inline src/components/DateSlider/index.js:126]: We lost the ternary -- what happens when `date.data` has length 0 or is null? Does `dateMarksFormat` protect against that?

**Review by saanobhaai — COMMENTED** [review]
Just two minor comments

File: src/components/Footer/index.js (hunk 1 above)
Comment (tienhoah) [reply src/components/Footer/index.js]: no problems

File: src/components/DateSlider/index.js (hunk 2 above)
Comment (tienhoah) [reply src/components/DateSlider/index.js:126]: added the counter to check

**Pull request merged**